.eslintrc.json

```diff
@@ -105,9 +105,13 @@
                 }
             }
         ],
-
-        // Todo: For each of these, investigate whether we want to enable them ✨
-        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": [
```

Comment:
If anything, I would probably prefer the settings on https://github.com/microsoft/TypeScript/pull/55139/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR81, as they match TypeScript's. ignoreRestSiblings is interesting, though.